auth-service/src/main/resources/templates/thymeleaf/set-password-form.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en" class="govuk-template " xmlns:th="http://www.thymeleaf.org">
+
+<head th:replace="~{fragments/template.html :: head (title='Delta | Set password')}">
+    <title>Delta | Set password</title>
+</head>
+
+<body class="govuk-template__body govuk-body">
+<a href="#main-content" class="govuk-skip-link" data-module="govuk-skip-link">Skip to main content</a>
+<header th:replace="~{fragments/template.html :: header}"></header>
+
+<!--/*@thymesVar id="deltaUrl" type="java.lang.String" */-->
+<!--/*@thymesVar id="message" type="java.lang.String" */-->
+<!--/*@thymesVar id="newPassword" type="java.lang.String" */-->
+<!--/*@thymesVar id="confirmPassword" type="java.lang.String" */-->
+
+<div class="govuk-width-container" id="setPasswordPage">
+    <main class="govuk-main-wrapper " id="main-content" role="main">
+        <h1 class="govuk-heading-xl">Set password</h1>
+        <div class="govuk-grid-row">
+            <div id="error" class="govuk-error-summary" th:if="${message != null}">
+                <h2 class="govuk-error-summary__title">There is a problem</h2>
+                <div class="govuk-error-summary__body" role="group"
+                     aria-labelledby="error-summary-heading-example-1" tabindex="-1">
+                    <ul class="govuk-list govuk-error-summary__list">
+                        <li>
+                            <div th:if="${ #strings.contains(message, 'Confirm password')}">
+                                <a href="#confirmPassword" th:text="${message}"></a>
+                            </div>
+                        </li>
+                        <li>
+                            <div th:unless="${ #strings.contains(message, 'Confirm password')}">
+                                <a href="#newPassword" th:text="${message}"></a>
+                            </div>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <p>Enter your password below. It must:<br/>
+                - be at least 12 characters<br/>
+                - not contain % &lt; or &gt;</p>
+            <form th:action="@{#}" method="post" autocomplete="off" id="set_password_form">
+                <div class="govuk-form-group"
+                     th:classappend="${message != null and !#strings.contains(message, 'Confirm password')} ? 'govuk-form-group--error' : ''">
+                    <label class="govuk-label" for="newPassword">New password</label>
+                    <div id="newPasswordError">
+                        <span class="govuk-visually-hidden">Error:</span>
+                        <p class="govuk-error-message"
+                           th:if="${message != null and !#strings.contains(message, 'Confirm password')}"
+                           th:text="${message}"></p>
+                    </div>
+                    <input id="newPassword" autocomplete="off" class="govuk-input govuk-!-width-two-thirds"
```

Comment:
```suggestion
                    <input id="newPassword" autocomplete="new-password" class="govuk-input govuk-!-width-two-thirds"
```
and I think the confirm box should be the same